### PR TITLE
References for N-Way Model Matching

### PR DESCRIPTION
**Pull request by PaulAtTUBS**
Hi Thomas,

these are the integrated references of Alex and Timo.

Greetings,
Paul

### CODE REVIEW

**Comment by tthuem** [comment]
Sorry, there have been multiple simultaneous changes recently. Could you take care of the merge conflicts please?

**Comment by PaulAtTUBS** [comment]
Ok no problem. I will do and comment here again when I am done with that.

**Comment by PaulAtTUBS** [comment]
I merged all your and @Subaro 's changes now.

**Comment by PaulAtTUBS** [comment]
By accident, I commited new versions of classifications.csv and citations.csv. If this is a problem, please let me know.

**Comment by PaulAtTUBS** [comment]
Hi Thomas,

I reverted the classifications.csv for now. I could not figure out yet why the newly generated one contains so many new lines.
You can decide if you want to merge now or wait for the new classifications.

Greetings
Paul

File: literature.bib [hunk 1]
```diff
@@ -6255,11 +6253,11 @@ @inproceedings{KKPS:ASE12
 }
 
 @book{BCW12,
-	author    = {Marco Brambilla and Jordi Cabot and Manuel Wimmer},
-	title     = {Model-Driven Software Engineering in Practice},
+	author = {Marco Brambilla and Jordi Cabot and Manuel Wimmer},
+	title = {Model-Driven Software Engineering in Practice},
```
Comment (tthuem) [inline literature.bib:6391]: Titles is missing brackets

**Pull request merged**